CMakeLists.txt

```diff
@@ -797,14 +807,9 @@ else()
 endif()
 
 # Detect all required libraries
-find_package(PNG REQUIRED)
-target_link_libraries(${PROJECT_NAME} PNG::PNG)
-
-find_package(Pixman REQUIRED)
-target_link_libraries(${PROJECT_NAME} PIXMAN::PIXMAN)
-
-find_package(fmt REQUIRED)
-target_link_libraries(${PROJECT_NAME} fmt::fmt)
+player_find_package(NAME PNG TARGET PNG::PNG REQUIRED)
+player_find_package(NAME Pixman TARGET PIXMAN::PIXMAN REQUIRED)
```

Comment:
I think the reason for the Android build error is this: Pixman enables the C language on Android for cpufeatures.c and due to weird CMake variable scoping rules the variables are not properly set in parent scope